src/user_data.rs

```diff
@@ -167,6 +172,14 @@ pub trait UserDataModule {
         self.get_user_stake_of_type(USER_STAKE_TOTALS_ID, UserStakeState::WithdrawOnly)
     }
 
+    fn validate_total_user_stake(&self, user_id: usize) -> Result<(), SCError> {
+        let user_total = self.get_user_total_stake(user_id);
+        if &user_total > &0 && &user_total < &self.settings().get_minimum_stake() {
```

Comment:
Because user_total is of type BigUint, does not implement Copy. Comparing such types directly consumes them, and we still need if for the next condition.
So, we just compare references to them. And &0 because if there is reference on one side, Rust requires the other too.

Reply:
My bad, they were unnecessary